src/truncated/exponential.jl

```diff
@@ -0,0 +1,19 @@
+#####
+##### Truncated exponential distribition
+#####
+
+function mean(d::Truncated{<:Exponential,Continuous})
+    θ = d.untruncated.θ
+    l, r = extrema(d)           # l is always finite
+    if isfinite(r)
+        if l == r
```

Comment:
shouldn't that be `\approx` instead of equality?

Reply:
I don't see why. It is just there to protect from the really degenerate case, the formula with the midpoint should work fine for `r ≈ l` (see the tests, where I check this).

Reply:
I think this might be fine. The only way it could go wrong is when  l - r is small but not zero and theta is unreasonable large, e.g. `l-r = eps()`, `theta = 1e308`

Reply:
You are right, I added a case for that. If CI passes, I wonder if this could be merged now (and also please tag a minor release).